PennMobile/GSR-Booking/Views/RoomFinderSelectionPanel.swift

```diff
@@ -83,21 +84,20 @@ struct RoomFinderSelectionPanel: View {
                         return
                     }
                     Task { @MainActor in
-                        let qb = GSRQuickBook()
                         do {
-                            try await qb.populateSoonestTimeslot(location: location, duration: duration, time: time)
+                            try await quickBook.populateSoonestTimeslot(location: location, duration: duration, time: time)
```

Comment:
Is there a reason we need to separate out `populateSoonestTimeslot()` and `quickBook()`?